docs/management/managing-indices.asciidoc

```diff
@@ -141,28 +202,41 @@ Alternatively, you can click the *Load JSON* link and define the mapping as JSON
     }
   }
 }
-----------------------------------
-
+----
++
 You can create additional mapping configurations in the *Dynamic templates* and
 *Advanced options* tabs. No additional mappings are required for this example.
 
-In the fourth step, define an alias named `logstash`.
+. Click *Next >*.
 
+*Step 5: Aliases*
+
+. Define an index alias named `my-index`:
++
 [source,js]
-----------------------------------
+----
 {
-  "logstash": {}
+  "my-index": {}
 }
-----------------------------------
+----
+
+. Click *Next >*.
+
+*Step 6: Review template*
```

Comment:
Because steps 3-6 are so short, combing these in one section.  And then leave out "Click Next" in every step because the user can figure that out.

**Step 2.  Add settings** (or whatever the appropriate term is)

1.   Define the component templates

     text about components

      They are optional.  Leave blank for this example.

2.  Define index settings.

    These are optional.  For this tutorial, leave this section blank.

3. Define a mapping that contains an object field named geo with a child geo-point field named coordinates:

    image and JSON

4.  Define an index alias named my-index:

5.  On the review page, check the summary, and if everything looks right click Create template

**Step 3.  Load the logs data**